source/modules/MEI.cmn.xml

```diff
@@ -438,6 +438,15 @@
   <classSpec ident="att.mNum.log" module="MEI.cmn" type="atts">
     <desc xml:lang="en">Logical domain attributes.</desc>
   </classSpec>
+  <classSpec ident="att.damp.log" module="MEI.cmn" type="atts">
+    <desc xml:lang="en">Logical domain attributes.</desc>
+    <classes>
+      <memberOf key="att.controlEvent"/>
+      <memberOf key="att.duration.additive"/>
+      <memberOf key="att.startEndId"/>
+      <memberOf key="att.timestamp2.log"/>
```

Comment:
```suggestion
      <memberOf key="att.startId"/>
      <memberOf key="att.timestamp.log"/>
```

Maybe adding an attribute that takes values from `data.STRINGNUMBER` to specify which string/chorus is to be damped?

Reply:
I suspect that membership in att.duration.additive will come up again later, but I'm fine with dropping it for now.

I don't think another attribute to indicate which string is to be damped is necessary.  That's covered in harp notation by the vertical position of the damp sign.  Probably in guitar notation as well, but I haven't checked.

![image](https://github.com/music-encoding/music-encoding/assets/5712877/502d7217-7f2a-4158-99eb-7baeb0aad11f)

The precise performed duration of each note can be handled by `@dur.ges`.